service/resource/kmskeyv2/resource.go

```diff
@@ -3,9 +3,14 @@ package kmskeyv2
 import (
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
```

Comment:
Could you pls put together all the gs imports in a separate block?

Reply:
I am not sure what do you mean. I thought we set first built-in go libraries, then external from the project, and finally packages from the same project). I have seen it in others files too, so I am bit confuse

Reply:
Yes the convention is the 3rd block has packages from the same project. So this looks correct to me.

Reply:
Indeed, didn't notice the external gs projects and assumed that all the gs imports came from this one, sorry about that :)